src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,56 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+
+    if (!Animal.alive) {
+      Animal.alive = [];
+    }
+    Animal.alive.push(this);
+  }
+
+  static removeFromAlive(animal) {
+    Animal.alive = Animal.alive.filter((a) => a !== animal);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    super(name, health);
+    this.hidden = false;
+
+    Animal.alive.push(this);
+  }
+
+  hide() {
+    this.hidden = true;
+  }
+
+  static removeFromAlive(animal) {
+    Animal.alive = Animal.alive.filter((a) => a !== animal);
+  }
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    super(name, health);
+
+    Animal.alive.push(this);
+  }
```

Comment:
```suggestion

```

Reply:
I can't understand what i have to do

Reply:
Somebody explain please what is wrong and leave comments to mistakes please

Reply:
![photo_2024-04-29_17-07-45](https://github.com/mate-academy/js_herbivores_and_carnivores/assets/120343956/c676d98c-3fce-4793-858d-c99996e9252f)

I don't see the difference